test/diff_test.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:pana/src/third_party/diff_match_patch/test.dart';
+
+void main() {
```

Comment:
Instead of this, each `<something>_test.dart` file should have its own `main()` method separately, that way they can be run only as a subset of the total test suite, and also run in parallel.

(Similar to license_filter_test.dart or license_util_test.dart).

Reply:
I think we should leave a comment, that this is because the actual tests are part of the `third_party/` folder...